pingcap/tidb-operator
# Support to customize tidb service port

### What problem does this PR solve?

Close #4511
### What is changed and how does it work?


### Code changes


### Tests



* Deploy cluster with operator v1.3.2, and upgrade operator. Check no rolling upgrade.
* Deploy cluster with `spec.tidb.service.port: 5000` and `spec.tidb.initializer.createPassword: true`.
* Deploy cluster with `spec.tidb.service.port: 5000`, and deploy `TidbInitializer` to initialize.
* Deploy cluster with `spec.tidb.service.port: 5000`, and deploy `Backup`


### Side effects


### Related changes


### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
Support to customize tidb service port
```
